main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       phone
+//  action 1:   sending call
+phone.sendingCall();
+//  action 2:   receving call
+phone.recevingCall();
+//  action 3:   taking picture
+phone.takingPicture();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       computer
+//  action 1:   activates wifi
+computer.activateWifi();
+//  action 2:   laoding
+computer.loading();
+//  action 3:   shuts down
+computer.shutdown();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       washing machine
+//  action 1:   washing
+washer.washing();
+//  action 2:   taking hot water
+washer.takingHotWater();
+//  action 3:   taking cold water
+washer.takingColdWater();
+//  4)         
+//  Noun:       traffic light
+//  action 1:   turns green
+trafficLight.turnsGreen();
+//  action 2:   turns yellow
+trafficLight.turnsYellow();
+//  action 3:   turns red
+trafficLight.turnsRed();
+//  5)         
+//  Noun:       midi keyboard
+//  action 1:   receiving midi signal
+midiKeyboard.recevingMidiSignal();
+//  action 2:   sending midi signal
+midiKeyboard.sendingMidiSignal();
+//  action 3:   shows CC  
+midiKeyboard.showCc();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.